### PR TITLE
Remove unused dependencies

### PR DESCRIPTION
**Pull request by magopian**

### CODE REVIEW

File: requirements/prod.txt [hunk 1]
```diff
@@ -61,7 +60,6 @@ mock==1.0.1
 monolith.client==0.9
 moz-addon-packager==1.0.1
 mozilla-logger==0.2
-mozpay==2.0.0
 m2secret==0.1.1
```
Comment (kumar303) [inline requirements/prod.txt]: you probably don't need m2secret either. That's for encryption/decryption we were doing in zamboni.

**Comment by kumar303** [comment]
r+wc

File: requirements/prod.txt (hunk 1 above)
Comment (diox) [inline requirements/prod.txt]: I wonder if olympia needs `m2crypto` then ? (in requirements/compiled.txt)

Comment (kumar303) [inline requirements/prod.txt]: It probably no longer needs M2Crypto. That would save you guys a lot of headache if you could remove that one.

**Pull request merged**